### ISSUE TITLE
[Entry]: WebCord

### ISSUE THREAD

**Issue by UncleEnk** [Discord-Client-Encyclopedia-Management/Discord3rdparties#53]
### Name

Webcord

### Link

https://github.com/SpacingBat3/WebCord

### Features

It tries to conform within the ToS as much as possible

### Language(s)

Typescript

### Development Status

On hiatus

**Comment by SpacingBat3** [comment]
> **Development Status**: On hiatus

The development state of WebCord is kinda complicated… I still want to bring **at least** security updates and bug fixes and therefore it should be considered as maintained. In fact, from the time I've announced that WebCord's development might slow down, I've worked on many bug fixes and prepared my client for newer APIs. Some smaller features were also announced and there's a little hope I'll implement some smaller features in the nearest future as well, but all of that matters on my free time.

The more accurate word for the current status would be a *Variable*, a such state that still seems to be active on the first glance, but some occasional breaks might occur in some time (depending on the maintainers' free time). Although that's not my software list and neither my interest how you treat a software in such state as WebCord currently is, I just wanted to examine a little the situation with a smaller hope that this project won't be misleading to anyone visiting that page and comparable to the projects that doesn't receive any updates for some time.
